src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonAPI.kt

```diff
@@ -343,4 +347,97 @@ object DungeonAPI {
             fun getByInventoryName(inventory: String) = entries.firstOrNull { it.inventory == inventory }
         }
     }
+
+    enum class DungeonPhase {
```

Comment:
you can embed the patterns directly into the enum. Look at Skyblock Guide Highlight as an example of how (Not a good example but the first that came to mind)